GUI/MainWindow.xaml.cs

```diff
@@ -256,12 +305,40 @@ private void UpdateOutputFolderTextbox()
                     where rnaSeqFastqCollection.Select(b => b.FilePath).All(f => f.StartsWith(possibleMatch, StringComparison.Ordinal))
                     select possibleMatch;
 
-                OutputFolderTextBox.Text = Path.Combine(Path.GetDirectoryName(MatchingChars.First()), @"$DATETIME");
+                OutputFolderTextBox.Text = Path.Combine(Path.GetDirectoryName(MatchingChars.First()));
             }
             else
             {
                 OutputFolderTextBox.Clear();
             }
         }
+
+        private void AddAFile(string filepath)
+        {
+            var theExtension = Path.GetExtension(filepath).ToLowerInvariant();
+            switch (theExtension)
+            {
+                case ".fa":
```

Comment:
How will we differentiate protein and genome fastas? Both of those are required.

Fastas can also be gunzipped (.gz), and so can gene models (.gtf.gz, .gff3.gz).

Instead of checking for two file endings at once, e.g. "fastq.gz," I'd recommend checking for compression first, and then trim that file extension off. See https://github.com/smith-chem-wisc/MetaMorpheus/blob/master/TaskLayer/MetaMorpheusTask.cs#L323

Please also add a collection for protein XML.

I'm curious how we're going to differentiate a reference gene model from an alternative one (e.g. PacBio or otherwise) from this menu. Would it just be a checkbox for the reference in the dataframe?